distrobox-create

```diff
@@ -33,8 +33,10 @@
 #	DBX_NON_INTERACTIVE
 #	DBX_SUDO_PROGRAM
 
-# Dont' run this command as sudo.
-if [ "$(id -u)" -eq 0 ]; then
+# Although running this script via sudo is not supported, we still want
+# to allow people to run it as root, using a login shell, and get rootful
+# containers, of course.
+if [ -n "${SUDO_USER}" ]; then
```

Comment:
Is this valid also for other "sudo" implementations like `doas`?

Reply:
I can't tell first hand but if I would guess I'd say no. But I'm willing to take a look at a couple of DOAS Linux ports/implementation and see if there's a similar mechanism in use, otherwise I think I know how we could go about being DOAS-aware.

Reply:
Seems like `doas` does something similar to sudo, injecting a `DOAS_USER` in the env

Reply:
Yes, it does. Which's going to make our life easier.